.gitignore

```diff
@@ -30,3 +30,4 @@ __config__.py
 .coverage
 .buildbot.patch
 .eggs/
+.idea/
```

Comment:
I suggest you remove this changes and use a local `.gitignore_global` instead. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore